src/components/editgrid/EditGrid.js

```diff
@@ -937,7 +937,9 @@ export default class EditGridComponent extends NestedArrayComponent {
           const errorContainer = rowContainer.querySelector('.editgrid-row-error');
 
           if (!rowValid) {
-            errorContainer.textContent = this.t('invalidRowError');
```

Comment:
So, this should be picked up with the following.

https://github.com/formio/formio.js/blob/master/src/i18n.js#L11

Why is this not registering correctly?  I feel like this is not fixing the actual bug here...